src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -168,7 +168,8 @@ def zap_lv(self, device):
         """
         lv = api.get_single_lv(filters={'lv_name': device.lv_name, 'vg_name':
                                         device.vg_name})
-        pv = api.get_single_pv(filters={'lv_uuid': lv.lv_uuid})
+        #pv = api.get_single_pv(filters={'lv_uuid': lv.lv_uuid})
```

Comment:
is this commented out line being left here intentionally?

Reply:
leftover from devel/testing